main.py

```diff
@@ -1,11 +1,29 @@
 from flask import Flask
+from flask import send_from_directory
+from werkzeug.routing import BaseConverter
 
-app = Flask(__name__)
 
-@app.route('/')
-def index():
-    return 'Index Page'
+app = Flask(__name__, static_url_path="", static_folder="scs-app/dist/scs-app/")
```

Comment:
Could u add comments in the file why we need a static folder

Reply:
Or on top of the file it may be a good idea to document the work so other people can pick it up as well 👍